src/sql/workbench/api/browser/mainThreadQueryEditor.ts

```diff
@@ -73,7 +73,7 @@ export class MainThreadQueryEditor implements MainThreadQueryEditorShape {
 			let editor = filteredEditors[0];
 			if (editor instanceof QueryEditor) {
 				let queryEditor: QueryEditor = editor;
-				queryEditor.runCurrentQuery();
+				queryEditor.runQuery();
```

Comment:
do we need a different API for run current query?  Is there back compat concerns?  Or was this just an oversight originally?

Reply:
I think it is an oversight originally, now I am introduce an optional parameter to maintain the backward compatibility.